### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by bmos**
**I have not tested this as I don't have access to the data + APIs.**

Just some friendly suggestions. Take them or leave them.
Sorry if the line length/formatting goes too far into the personal preference realm, if so I can resubmit without that one.

Next steps I'd suggest would be adding docstrings throughout.

### CODE REVIEW

File: ListManagement/processNewMembers.py [hunk 1]
```diff
@@ -195,59 +236,73 @@ def setupEmail() -> EmailAPI.EmailAccount:
...
+        datetime.datetime.now() - datetime.timedelta(days=10),
+        expectedFileName=Constants.EXPECTED_LIST_ATTACHMENT_NAME,
+    )
+
     if os.path.exists(Constants.DOWNLOAD_LIST_PATH):
         logging.info("Found old downloaded list, deleting")
         os.remove(Constants.DOWNLOAD_LIST_PATH)
 
-    logging.info(f"Unzipping downloaded list to {Constants.DOWNLOAD_LIST_PATH}")
+    logging.info("Unzipping downloaded list to %s", Constants.DOWNLOAD_LIST_PATH)
```
Comment (matorin57) [inline ListManagement/processNewMembers.py:257]: Just curious here as I'm not too aware of the semantic difference between f-strings and passing in arguments. Is there a semantic difference or is this a style/linter change?

**Comment by matorin57** [comment]
Overall like the suggestions. I do have some specific comments however one of them could be a line length thing. I personally don't like line length guidelines too much as I've read some C++ where the line length requirement really messed with me. However I'm also normally in the Obj-C world which is very verbose and has very long lines (even when formatting to not have them).

File: ListManagement/processNewMembers.py (hunk 1 above)
Comment (bmos) [reply ListManagement/processNewMembers.py:257]: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

By passing the strings as args, the interpolation only occurs when the log message is posted. It saves processing time when not running with info messages enabled.

It's also better for log compilers but I don't know how relevant that is here.

**Comment by bmos** [comment]
I'll roll that one back, in looking it over this morning I tend to agree it's a mixed result

**Pull request merged**